src/po/zh_TW.po

```diff
@@ -1,5275 +1,9128 @@
-msgstr "E194: �S�� '#' �i��N���ɦW"
+#: ex_cmds.c:7669
+msgid "E612: Too many signs defined"
+msgstr "E612: 已定義太多 signs"
 
+#: ex_cmds.c:7750
+#, c-format
+msgid "E239: Invalid sign text: %s"
+msgstr "E239: 不正確的 sign 文字: %s"
+
+#: ex_cmds.c:7781 ex_cmds.c:7979
+#, c-format
+msgid "E155: Unknown sign: %s"
+msgstr "E155: 不正確的 sign: %s"
+
+#: ex_cmds.c:7814
+msgid "E159: Missing sign number"
+msgstr "E159: 缺少 sign number"
+
+#: ex_cmds.c:7908
+#, c-format
+msgid "E158: Invalid buffer name: %s"
+msgstr "E158: 緩衝區名稱錯誤: %s"
+
+#: ex_cmds.c:7937
+msgid "E934: Cannot jump to a buffer that does not have a name"
+msgstr "E934: 無法前往沒有名稱的緩衝區"
+
+#: ex_cmds.c:7952
+#, c-format
+msgid "E157: Invalid sign ID: %ld"
+msgstr "E157: Sign ID 錯誤: %ld"
+
+#: ex_cmds.c:7992
+#, c-format
+msgid "E885: Not possible to change sign %s"
+msgstr "E885: 不可能變更符號 %s"
+
+#: ex_cmds.c:8030
+msgid " (NOT FOUND)"
+msgstr " (找不到)"
+
+#: ex_cmds.c:8032
+msgid " (not supported)"
+msgstr " (不支援)"
+
+#: ex_cmds.c:8150
+msgid "[Deleted]"
+msgstr "[已刪除]"
+
+#: ex_cmds.c:8516
+msgid "No old files"
+msgstr "沒有舊檔"
+
+#: ex_docmd.c:571
+msgid "Entering Ex mode.  Type \"visual\" to go to Normal mode."
+msgstr "進入 Ex 模式. 輸入 \"visua\" 以回到正常模式."
+
+#: ex_docmd.c:621
+msgid "E501: At end-of-file"
+msgstr "E501: 已到檔案結尾"
+
+#: ex_docmd.c:724
+msgid "E169: Command too recursive"
+msgstr "E169: 命令遞迴層數過多"
+
+#: ex_docmd.c:1317
+#, c-format
+msgid "E605: Exception not caught: %s"
+msgstr "E605: 未攔截的例外： %s"
+
+#: ex_docmd.c:1403
+msgid "End of sourced file"
+msgstr "命令檔結束"
+
+#: ex_docmd.c:1404
+msgid "End of function"
+msgstr "函式結尾"
+
+#: ex_docmd.c:2300
+msgid "E464: Ambiguous use of user-defined command"
+msgstr "E464: 使用者定義的命令會混淆"
+
+#: ex_docmd.c:2315
+msgid "E492: Not an editor command"
+msgstr "E492: 不是編輯器的命令"
+
+#: ex_docmd.c:2435
+msgid "E493: Backwards range given"
+msgstr "E493: 指定了向前參考的範圍"
+
+#: ex_docmd.c:2439
+msgid "Backwards range given, OK to swap"
+msgstr "指定了向前參考的範圍，OK to swap"
+
+#: ex_docmd.c:2503
+msgid "E494: Use w or w>>"
+msgstr "E494: 請使用 w 或 w>>"
+
+#: ex_docmd.c:3126
+msgid "E943: Command table needs to be updated, run 'make cmdidxs'"
+msgstr "E943: 指令表格需要更新，請執行 'make cmdidxs'"
+
+#: ex_docmd.c:4703
+msgid "E319: Sorry, the command is not available in this version"
+msgstr "E319: 抱歉, 本命令在此版本中沒有實作"
+
+#: ex_docmd.c:5690
+msgid "1 more file to edit.  Quit anyway?"
+msgstr "還有一個檔案未編輯. 確定要離開？"
+
+#: ex_docmd.c:5694
+#, c-format
+msgid "%d more files to edit.  Quit anyway?"
+msgstr "還有 %d 個檔案未編輯. 確定要離開？"
+
+#: ex_docmd.c:5701
+msgid "E173: 1 more file to edit"
+msgstr "E173: 還有一個檔案未編輯"
+
+#: ex_docmd.c:5703
+#, c-format
+msgid "E173: %ld more files to edit"
+msgstr "E173: 還有 %ld 個檔案未編輯"
+
+#: ex_docmd.c:5796
+msgid "E174: Command already exists: add ! to replace it"
+msgstr "E174: 命令已經存在, 請使用 ! 強制重新定義"
+
+#: ex_docmd.c:5960
+msgid ""
+"\n"
+"    Name        Args       Address   Complete  Definition"
+msgstr ""
+"\n"
+"\t名稱\t\t參數\t\t位址\t自動完成\t定義"
```

Comment:
Isn't it better to use spaces instead of `\t` like the original message?
(Of course it is okay if the translated message is properly aligned, though.)

Reply:
No. Use \t to align better than SPACE because the bytes of English isn't same as Chinese.

Reply:
Did you confirmed it by executing the `:command` command?
The output of the `:command` is language-independent except the first line.